webhook/main.go

```diff
@@ -90,6 +93,18 @@ func main() {
 	var shutdownChan = make(chan os.Signal, 1)
 	signal.Notify(shutdownChan, syscall.SIGTERM)
 
+	// Setup health checks
+	if err := mgr.AddHealthzCheck("healthz", healthz.Ping); err != nil {
+		log.Error(err, "Unable to set up health check")
+		os.Exit(1)
+	}
+
+	// Setup health checks
+	if err := mgr.AddReadyzCheck("readyz", healthz.Ping); err != nil {
+		log.Error(err, "Unable to set up health check")
```

Comment:
```suggestion
		log.Error(err, "Unable to set up ready check")
```

Reply:
:woman_facepalming: good catch :smile: fixed! (same with the ones below)